### DISCUSSION TITLE
String pattern formatting options?

### DISCUSSION THREAD

**Discussion by leemcmullen** [barvian/number-flow#76] [Ideas]
Hey 👋 This is an awesome lib ❤️

I wondered if there's support in the pipeline for configuring formatting via string pattern options, similar to what you get with libs like [numerable](https://github.com/gastonmesseri/numerable)? 

~~E.g. if my number is 4,558,667,268.98 then I'd love numberflow to be able to output 4.56 B like this: `<NumberFlow value={bigNumber} format={'0.00 a'} />`. I don't think that's possible out of the box in a simple fashion at the moment.~~

Ignore this example, I think that's possible using `{notation: 'compact'}`, but I suppose the question still stands. I think it would still be really useful to be able to provide string formatting options instead of the Intl.NumberFormat() options 👍

**Comment by barvian** [comment]
Hi 👋, thanks for the discussion. Is there a certain format you had in mind that Intl.NumberFormat can't handle?
